Python/optimizer_bytecodes.c

```diff
@@ -138,6 +142,24 @@ dummy_func(void) {
         }
     }
 
+    op(_CHECK_ATTR_CLASS, (type_version/2, owner -- owner)) {
+        assert(type_version);
+        if (sym_is_a_class(owner) &&
+            sym_matches_type_version(owner, type_version)) {
+            REPLACE_OP(this_instr, _NOP, 0, 0);
+        } else {
+            sym_set_is_a_class(owner);
+            PyTypeObject *type = _PyType_LookupByVersion(type_version);
+            if (type) {
+                if (sym_set_type_version(owner, type_version)) {
```

Comment:
Likewise this is incorrect, as this set's the type version of the symbol's class.